src/marlin_stubs/sdcard/M20-M30_M32-M34.cpp

```diff
@@ -119,7 +120,17 @@ void GcodeSuite::M29() {
 
 // M30 - Delete a file on the SD card
 void GcodeSuite::M30() {
-    // TODO
+    char filepath[FILE_PATH_MAX_LEN] = "/usb/";
```

Comment:
I'd maybe suggest using our StringBuilder for this, though your code is fine in this regard. Though theoretically, we could have a wrong file deleted if the string arg happens to get cropped.

Reply:
Yeah, good idea.